Azure/azure-key-vault/_meta/fields.yml

```diff
@@ -0,0 +1,86 @@
+azure.key_vault.correlation_id:
+  description: The correlation id of the key vault operation
+  name: azure.key_vault.correlation_id
+  type: keyword
+
+azure.key_vault.identity.claim.appid:
+  description: The application id of the identity
+  name: azure.key_vault.identity.claim.appid
+  type: keyword
+
+azure.key_vault.properties.enablePurgeProtection:
+  description: Indicates whether the key vault is enabled for purge protection
+  name: azure.key_vault.properties.enablePurgeProtection
+  type: boolean
+
+azure.key_vault.properties.enableRbacAuthorization:
+  description: Indicates whether the key vault is enabled for role-based access control
+    authorization
+  name: azure.key_vault.properties.enableRbacAuthorization
+  type: boolean
+
+azure.key_vault.properties.enableSoftDelete:
+  description: Indicates whether the key vault is enabled for soft delete
+  name: azure.key_vault.properties.enableSoftDelete
+  type: boolean
+
+azure.key_vault.properties.enabledForDeployment:
+  description: Indicates whether the key vault is enabled for deployment
+  name: azure.key_vault.properties.enabledForDeployment
+  type: boolean
+
+azure.key_vault.properties.enabledForDiskEncryption:
+  description: Indicates whether the key vault is enabled for disk encryption
+  name: azure.key_vault.properties.enabledForDiskEncryption
+  type: boolean
+
+azure.key_vault.properties.enabledForTemplateDeployment:
+  description: Indicates whether the key vault is enabled for template deployment
+  name: azure.key_vault.properties.enabledForTemplateDeployment
+  type: boolean
```

Comment:
I doubt about the usefulness of these fields.
I suggest removing them
```suggestion
```
What do you think @Adamowoc?